include/hip/devicelib/double_precision/dp_math.hh

```diff
@@ -47,6 +78,41 @@ static inline __device__ long int lround(double x) {
 static inline __device__ long long int llrint(double x) { return lrint(x); }
 static inline __device__ long long int llround(double x) { return lround(x); }
 
+static inline __device__ double nearbyint(double x) {
+  return __builtin_nearbyint(x);
+}
+
+namespace std {
+// Clang does provide device side std:: functions via HIP include
+// wrappers but, alas, the wrappers won't compile on CHIP-SPV due to
+// presence of AMD specific built-ins.
+using ::acos;
+using ::asin;
+using ::atan;
+using ::ceil;
+using ::copysign;
+using ::cos;
+using ::cosh;
+using ::erf;
+using ::erfc;
+using ::exp;
+using ::expm1;
+using ::floor;
+using ::lgamma;
+using ::log;
+using ::log10;
+using ::log1p;
+using ::log2;
+using ::nearbyint;
+using ::nextafter;
+using ::sin;
+using ::sinh;
+using ::sqrt;
+using ::tan;
+using ::tanh;
+using ::trunc;
+} // namespace std
+
 // __device__ double acos(double x)
 // __device__​ double acosh ( double  x )
 // __device__​ double asin ( double  x )
```

Comment:
Placeholders should be removed and it would be preferable that the line order is maintained. These placeholders were made by copying line-by-line from the API definition https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html

Reply:
Removed and reordered declarations.